scripts/commands.js

```diff
@@ -80,7 +80,9 @@ program
 
 program
   .command('upload')
+  .option('--target_os <target_os>', 'target OS')
   .option('--target_arch <target_arch>', 'target architecture')
+  .option('--target_apk_base <target_apk_base>', 'target Android OS apk (classic, modern, mono)', 'classic')
```

Comment:
it would be much safer and more reliable to read these values out of the src/out/{build_type}/args.gn file instead of duplicating them in every command

Reply:
what's the best way to do that?